projects/layout/components/chip-group/chip-group.styles.less

```diff
@@ -0,0 +1,40 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+tui-chip-group,
+[tuiChipGroup] {
+    display: flex;
+    flex-wrap: wrap;
+
+    tui-chip,
```

Comment:
Can we change it to direct children? Might be helpful for thumbnail card, badges, not just chips. Maybe even rename it to `ItemGroup` as in `ItemsWithMore`?